src/components/directory-content.js

```diff
@@ -719,60 +753,37 @@ const DirectoryContent = () => {
         setSelectedUuids(new Set());
     }, [handleError, currentChildren, currentChildrenRef]);
 
-    const contextualMixPolicies = {
-        BIG: 'GoogleMicrosoft', // if !selectedUuids.has(selected.Uuid) deselects selectedUuids
-        ZIMBRA: 'Zimbra', // if !selectedUuids.has(selected.Uuid) just use activeElement
-        ALL: 'All', // union of activeElement.Uuid and selectedUuids (actually implemented)
-    };
-    let contextualMixPolicy = contextualMixPolicies.ALL;
-
-    const getSelectedChildren = (mayChange = false) => {
-        let acc = [];
-        let ctxtUuid = activeElement ? activeElement.elementUuid : null;
-        if (activeElement) {
-            acc.push(
-                Object.assign(
-                    {
-                        subtype:
-                            childrenMetadata[activeElement.elementUuid]
-                                ?.subtype,
-                    },
-                    activeElement
-                )
-            );
-        }
-
-        if (selectedUuids && currentChildren) {
-            if (
-                contextualMixPolicy === contextualMixPolicies.ALL ||
-                ctxtUuid === null ||
-                selectedUuids.has(ctxtUuid)
-            ) {
-                acc = acc.concat(
+    const getSelectedChildren = () => {
+        let selectedChildren = [];
+        if (currentChildren?.length > 0) {
+            // Adds the previously selected elements
+            if (selectedUuids?.size) {
+                selectedChildren = selectedChildren.concat(
                     currentChildren
                         .filter(
                             (child) =>
                                 selectedUuids.has(child.elementUuid) &&
                                 child.elementUuid !== activeElement?.elementUuid
                         )
-                        .map((child2) => {
-                            return Object.assign(
-                                {
-                                    subtype:
-                                        childrenMetadata[child2.elementUuid],
-                                },
-                                child2
-                            );
+                        .map((child) => {
+                            return {
+                                subtype: childrenMetadata[child.elementUuid],
+                                ...child,
+                            };
                         })
                 );
-            } else if (
-                mayChange &&
-                contextualMixPolicy === contextualMixPolicies.BIG
-            ) {
-                setSelectedUuids(null);
+            }
+
+            // Adds the active element
+            if (activeElement) {
+                selectedChildren.push({
+                    subtype:
```

Comment:
I prefer : 
```
{
    ...activeElement,
    subtype: childrenMetadata[activeElement.elementUuid]?.subtype,
}
```
because it clearly says that we overwrite any existing or not `subtype` value in `activeElement`, otherwise you could think reading the code that `subtype` has the value of `childrenMetadata[activeElement.elementUuid]?.subtype` but if this key is present in the `activeElement` object then it will overwrite and prevail.

Reply:
Done